containers/runtime/xds/config/config.go

```diff
@@ -459,3 +453,35 @@ func (t *TestResource) UpdateEndpoint(snap cache.Snapshot) error {
 	}
 	return nil
 }
+
+// SocketListenerOnly takes a pointer of a snapshot, and returns only the socket listeners.
+// This function is used for Proxied test since api_listneners which are used for the
+// non-proxed test can not be validated by Envoy causing the entire resources slices not to be
+// registed.
+func SocketListenerOnly(snap *cache.Snapshot) error {
```

Comment:
Nit: Function names should be verb phrases.

Reply:
I make it `IncludeSocketListenerOnly`. Thanks.